tests/test_evaluate.py

```
@@ -15,6 +19,20 @@
+    plot_custom_instances,
+    predictions_viz,
+    predictions_viz_multiple_files,
+    predictions_viz_from_sleap_files,
+    visualize_predictions_from_artifacts,
+    get_runs_by_sweep_name_pattern,
+    fetch_metrics_from_sweep_pattern,
+    group_sweep_runs_retroactively,
+    find_and_evaluate_recent_sweeps,
+)
+from tests.fixtures import (
+    environment_config,
```

Ruff (0.12.2)

`tests.fixtures.environment_config` imported but unused

Remove unused import

(F401)